neo/Implementations/Wallets/peewee/UserWallet.py

```diff
@@ -79,19 +79,43 @@ def Open(path, password):
 
     @staticmethod
     def Create(path, password):
+        """
+        Create a new user wallet.
+
+        Args:
+            path (str): A path indicating where to create or open the wallet i.e. "/Wallets/mywallet".
+            password (str): a 10 characters minimum password to secure the wallet with.
+
+        Returns:
+             UserWallet: a UserWallet instance.
+        """
         wallet = UserWallet(path=path, passwordKey=password, create=True)
         wallet.CreateKey()
         return wallet
 
     def CreateKey(self, prikey=None):
+        """
+        Create a KeyPair and store it encrypted in the database.
+
+        Args:
+            private_key (iterable_of_ints): (optional) 32 byte private key.
+
+        Returns:
+            KeyPair: a KeyPair instance.
+        """
         account = super(UserWallet, self).CreateKey(private_key=prikey)
-        self._keys[account.PublicKeyHash.ToBytes()] = account
```

Comment:
I don't think you can delete this line. `Wallet.py` uses `self._keys`

Reply:
the super class does the exact same thing from which you can access it:
https://github.com/CityOfZion/neo-python/blob/master/neo/Wallets/Wallet.py#L274

the `neo` C# actually does the same thing as well
https://github.com/neo-project/neo/blob/master/neo/Implementations/Wallets/EntityFramework/UserWallet.cs#L102-L107

Reply:
Right! 👍